test/rendering-test/webpack.config.test-rendering.js

```diff
@@ -25,16 +25,20 @@ module.exports = {
 
   resolve: {
     alias: {
-      'deck.gl': resolve('./src')
+      'deck.gl': resolve('../../src')
     }
   },
 
   module: {
     rules: [
       {
-        include: [resolve('./src')],
+        include: [resolve('../../src')],
         loader: 'transform-loader',
         options: 'brfs-babel'
+      },
+      {
+        test: /\.png$/,
```

Comment:
I'm not exactly sure how these loaders and brfs works... will need to sync with you when you have time